src/engine.cpp

```diff
@@ -1,3 +1,48 @@
 #include "engine.h"
-
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <random>
 // Create definitions of the RandomEngine class here
+// Methods:
+void RandomEngine::setLength(uint8_t l){
+    length = l;
+}
+uint8_t RandomEngine::getLength(){
+    return length;
+}
+void RandomEngine::setSymbol(bool s){
+    symb = s;
+}
+bool RandomEngine::getSymbolStatus(){
+    return symb;
+}
+std::string RandomEngine::getString(){
+    int mod=62;
+    if(getSymbolStatus()){
```

Comment:
Well, Since we are in the class function, we can access members directly, which will be more efficient than calling this function. It just adds a function in the call stack of the program.